flash/core/model.py

```diff
@@ -188,3 +210,111 @@ def on_save_checkpoint(self, checkpoint: Dict[str, Any]) -> None:
 
     def configure_finetune_callback(self):
         return []
+
+    ### THE FOLLOWING IS A POC FOR DISTRIBUTED PREDICTION
+    def on_predict_start(self):
```

Comment:
@tchaton  does it make sense to have a hook like that (I think we need to revisit lightning hooks in general for all stages)

Reply:
Where is it called ? I guess we could add hook for predict. Need a bit more exploration there.

Reply:
Similar to on_fit_start, would be called immediately after the Trainer.predict was called